### ISSUE TITLE
Is it possible to use different cameras that aren't USB?

### ISSUE THREAD

**Issue by DanPugly** [iizukanao/picam#109]
I was going to ask if I can use USB cameras but then I saw another post before, but my question is can I use different cameras that aren't USB such as this one maybe?: https://www.amazon.com/Arducam-Megapixels-OV5647-Camera-Raspberry/dp/B012ETE75I/ref=sr_1_7?ie=UTF8&qid=1529670784&sr=8-7&keywords=pi+camera

**Comment by umolac** [comment]
Well, since it uses CSI, and has specification really similar to Camera V1, I don't think that shouldn't be possible

**Comment by DanPugly** [comment]
yeah but like in principle, I took the first camera that was similar to the raspberry pi cam on Amazon :D
so it shouldn't have any problems...

**Comment by iizukanao** [comment]
Sorry, but USB cameras are not supported.

**Comment by DanPugly** [comment]
I didnt mean a USB cam i meant a cemra that uses the same connection as the
official raspberry pi cam?

**Comment by iizukanao** [comment]
@DanPugly Oh, sorry for misunderstanding. I think that Arducam camera works because it uses the same interface (CSI connector and OV5647 image sensor) as Raspberry Pi Camera Module v1.

**Comment by DanPugly** [comment]
Ok thanks :D

**Comment by DanPugly** [comment]
Is the official camera v2 supported?

On Jun 24, 2018 10:00, "Pugly" <dushomirsky@gmail.com> wrote:

Ok thanks :D

**Comment by iizukanao** [comment]
@DanPugly Yes, Camera Module V2 is supported.
